models/organization/org.go

```diff
@@ -356,13 +356,23 @@ func DeleteOrganization(ctx context.Context, org *Organization) error {
 		return fmt.Errorf("%s is a user not an organization", org.Name)
 	}
 
+	teams, err := FindOrgTeams(ctx, org.ID)
+	if err != nil {
+		return fmt.Errorf("FindOrgTeams: %v", err)
+	}
+	for _, team := range teams {
+		if err := db.DeleteBeans(ctx, &TeamInvite{TeamID: team.ID}); err != nil {
+			return fmt.Errorf("DeleteBeans: %v", err)
```

Comment:
I mean, I need to bend the rules here but the context requires that:
```suggestion
			return fmt.Errorf("DeleteBeans (TeamInvite): %v", err)
```
Otherwise, there's a possibility that you can't tell which `db.DeleteBeans` failed afterwards.